src/exampleTypes/index.js

```diff
@@ -40,7 +40,15 @@ export default ({
   let Components = {
     Facet: defaultProps({ TextInput, Checkbox, RadioList })(Facet),
     Number: defaultProps({ NumberInput, Button })(Number),
-    Date: defaultProps({ DateInput })(Date),
+    Date: defaultProps(
+      DateInput
+        ? { DateInput, RadioList, Select }
+        : {
+            DateInput: defaultProps({ type: 'date' })(Input),
```

Comment:
it's simpler to just put this in the default for DateInput like we did for NumberInput, etc